core/server/web/members/app.js

```diff
@@ -11,6 +13,16 @@ module.exports = function setupMembersApp() {
     debug('Members App setup start');
     const membersApp = express('members');
 
+    // Body parsing
+    membersApp.use(bodyParser.json({limit: '1mb'}));
+    membersApp.use(bodyParser.urlencoded({extended: true, limit: '1mb'}));
+
+    // Query parsing
+    membersApp.use(boolParser());
+
+    // send 503 json response in case of maintenance
+    membersApp.use(shared.middlewares.maintenance);
+
```

Comment:
@ErisDS Wanted to add the `bodyParser` middleware for update endpoint, but since we are using these common middlewares across all our apps so extended them here with `boolParser` and `maintenance` check as well